cppapi/client/helpers/DeviceProxyHelper.h

```diff
@@ -424,7 +424,7 @@ namespace Tango
 		//  Overloaded commands to  avoid usage of DevVarXX ARRAY for argout
 		//---------------------------------------------------------------------------
 		template <class _IN>
-			void internal_command_inout (const std::string& cmd_name, const _IN& argin, DevVarDoubleStringArray* argout,std::string file= __FILE__, int line= __LINE__)
+			void internal_command_inout (const std::string& TANGO_UNUSED(cmd_name), const _IN& TANGO_UNUSED(argin), DevVarDoubleStringArray* TANGO_UNUSED(argout), std::string file= __FILE__, int line= __LINE__)
```

Comment:
Should we maybe remove these unused parameters at all?

Reply:
`TANGO_UNUSED` instead of just skipping the name has the advantage that we still have the name which tells us what is supposed to be passed in.

But I don't have a strong opinion here, so I can also change it.

Reply:
I vote for using TANGO_UNUSED for this specific example so we have a better understanding of the meaning of the arguments.
Sometimes we can get also this kind of warnings in methods where some arguments are used only when compiling with some specific compiler versions or with some specific compiler flags (-Dxxxx).
In this case it is also good to use TANGO_UNUSED.